lib/put-setting-api.js

```diff
@@ -3,74 +3,74 @@
 const assert = require('assert');
 const { API } = require('@janiscommerce/api');
 const { struct } = require('@janiscommerce/superstruct');
+const path = require('path');
 
 const DefinitionFetcher = require('./definition-fetcher');
-const ClientSettingsModel = require('./client-settings-model');
 
+const clientModelPath = path.join(process.cwd(), process.env.MS_PATH || '', 'models', 'client');
 module.exports = class PutSettingApi extends API {
 
+	get model() {
+
+		try {
+			// eslint-disable-next-line global-require, import/no-dynamic-require
+			const ClientModel = require(clientModelPath);
+			return new ClientModel();
+		} catch(error) {
+			this.setCode(400);
+			throw new Error(`Invalid Model Client. Must be in ${clientModelPath}.`);
```

Comment:
El mensaje puede ser un poco confuso ya que no es que se este validando el modelo como tal, sino que cuando lo fue a buscar no estaba en el path que debería. Capaz seria mas claro algo como 'Model Client not found. Must be...'